src/components/SocialButtons/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Divider } from 'semantic-ui-react';
+
+export const SocialButtons = () => (
```

Comment:
you do not have to export here as it is exported at the bottom

Reply:
okay